dGame/dComponents/RacingControlComponent.cpp

```diff
@@ -374,8 +374,11 @@ void RacingControlComponent::HandleMessageBoxResponse(Entity* player, int32_t bu
 		data->collectedRewards = true;
 
 		// Calculate the score, different loot depending on player count
-		const auto score = m_LoadedPlayers * 10 + data->finished;
-
+		auto playersRating = m_LoadedPlayers * 10;
+        if (m_SoloRacing) {
+			playersRating *= 2;
+		}
+        const auto score = playersRating * 10 + data->finished;
```

Comment:
Why is it multiplying by 10 twice now?

Reply:
Easiest fix I could find that doesn't require changing lmis

Reply:
though this should probably check if you are solo first

Reply:
Should be up to par with your requests now